css/styles.css

```diff
@@ -234,6 +251,14 @@ header .menu-bars:hover {
   color: #6b778c;
 }
 
```

Comment:
You should remove this line of code from your codebase since it's not useful. Leaving unused code in your codebase makes it harder to read and reason about